device/pcie/utils.hpp

```diff
@@ -47,7 +47,7 @@ tenstorrent_get_device_info get_pcie_device_info(int device_fd) {
 std::uint32_t get_pcie_info(int device_id, const std::string &info_needed) {
     // Get PCIe device info through iotcl
     int device_fd = find_device(device_id);
-    static const tenstorrent_get_device_info device_info = get_pcie_device_info(device_fd);
+    auto device_info = get_pcie_device_info(device_fd);
```

Comment:
If you choose to move this to pci_device as part of this PR (it's okay if you don't, we can improve it later), then this is unnecessary since you already have it in "this->info"

Reply:
I created https://github.com/tenstorrent/tt-umd/issues/147 to remove this utils.hpp